docker/Dockerfile

```diff
@@ -57,20 +58,72 @@ ENV LD_LIBRARY_PATH "/usr/local/lib:$LD_LIBRARY_PATH"
 # install KINC
 WORKDIR /opt
 
-ADD https://api.github.com/repos/SystemsGenetics/KINC/git/refs/heads/${KINC_REVISION} kinc-version.json
-RUN git clone -q https://github.com/SystemsGenetics/KINC.git \
-	&& mkdir -p KINC/build \
-	&& cd KINC/build \
-	&& git checkout -q ${KINC_REVISION} \
-	&& qmake ../src/KINC.pro GUI=no \
-	&& make -s -j $(nproc) \
-	&& make -s qmake_all \
-	&& make -s install
+# For older version of KINC we need to use qmake. For newer versions after 3.3.x we
+# can use the make file in the root directory of KINC.
+RUN if [ "${KINC_REVISION}" = "v3.3.x" ] || [ "${KINC_REVISION}" = "v3.2.x" ] ; then \
+        git clone -q https://github.com/SystemsGenetics/KINC.git \
+        && cd KINC/build \
+    	&& git checkout-q ${KINC_REVISION} \
```

Comment:
Missing a space between checkout and -q

Reply:
Good catch. I forgot to test a v3.3.x build, which is why I didn't catch these. I'll make sure I build one of these..  

This has been fixed in the latest push to the branch